text/0006-BundleContext-Install-API/0006-BundleContext-Install-API.md

```diff
@@ -0,0 +1,138 @@
+- Start Date: 2020-06-09
+- RFC PR: (in a subsequent commit to the PR, fill me with the PR's URL)
+- CppMicroServices Issue: (fill me with the URL of the CppMicroServices issue that necessisated this RFC, otherwise leave blank)
+
+# Add BundleContext::InstallBundles API
+
+## Summary
+
+Provide a api to initialize the bundle registry with a bundle manifest
+
+## Motivation
+
+Allows users of the system to inject bundle manifests into the registry directly. We will be able to use this mechanism to speed up initial install of very large numbers of bundles.
+
+## Detailed design
+
+![InstallBundlesSequence](./InstallBundlesSequence.svg)
```

Comment:
link doesn't work. take a look at https://raw.githubusercontent.com/CppMicroServices/rfcs/master/text/0003-declarative-services.md for an example of how .svg files are added to a document.